src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx

```diff
@@ -45,54 +46,79 @@ If your application is [instrumented with <InlinePopover type="browser"/>](/docs
+    
+    The `features` object should contain an entry for each feature initialized by the agent, including an `ajax` entry. 
+    
+    Verify this ajax object contains a boolean `enabled` of `true` and `featAggregate` entry. 
+    
+    If any of these items are not true, the AJAX instrumentation may not have initialized due to a configuration change in New Relic on the browser entity or possibly a manual change to the `NREUM.init` object that was copy/pasted into the HTML.
+  
+  <Callout variant="important">
+    `newrelic.initializedAgents` is the accepted property to check for agent existence on a page, unless you are using an old or an unsupported agent version.
+  </Callout>
+  </Step>
+  
+  <Step>
+  
+    ## Verify the deny list configuration
 
-  <Collapser
-    id="verify-config"
-    title="2. Verify the deny list configuration."
-  >
     The agent can be configured to ignore certain domains and paths when creating [AjaxRequest Events](/attribute-dictionary/?event=AjaxRequest).
 
     Use your browser's dev console to check the current AJAX deny list rules by running `newrelic.init`. Look for a property `ajax` containing an array called `deny_list`.
 
     If you see entries in the deny list that are unfamiliar or causing AJAX calls to be filtered, review the [Filter AjaxRequest Events](/docs/browser/new-relic-browser/configuration/filter-ajax-request-events/) documentation for further guidance.
-  </Collapser>
+  </Step>
+
+  <Step>
+
+    ## Verify network access
 
-  <Collapser
-    id="verify-network-access"
-    title="3. Verify network access."
-  >
     If the object is properly instrumented, try triggering an AJAX call in your application while monitoring network traffic in the browser's developer tools. Wait up to one minute, and look for a call to `bam.nr-data.net/jserrors` with an `xhr` parameter. If the call fails, check for network issues.
 
     If you don't see this call, if it fails with an error not related to network access, or if it succeeds but you still aren't seeing data, continue through the troubleshooting documentation.
-  </Collapser>
-</CollapserGroup>
 
-If your requests use JSONP, see requirements and notes on functionality below:
+  </Step>
+
+  <Step>
 
-<CollapserGroup>
-  <Collapser
-    id="jsonp"
-    title="JSONP requirements"
-  >
-    If your requests use [JSONP](https://en.wikipedia.org/wiki/JSONP), these requests will not appear on the [AJAX UI page](/docs/browser/new-relic-browser/browser-pro-features/ajax-page-identifying-time-consuming-calls). However, you can view them as assets within [session traces](/docs/browser/new-relic-browser/browser-pro-features/session-traces-exploring-webpages-life-cycle). If using SPA monitoring, you can view them on the <DNT>**Breakdown**</DNT> tab of the <DNT>**Page views**</DNT> page.
+  If your requests use JSONP, see requirements and notes on functionality below:
```

Comment:
@adutta-newrelic I think you could drop this sentence, and it wouldn't hurt the meaning of the section.